src/components/NavBar/style.scss

```
@@ -0,0 +1,27 @@
+@use '@/style/package' as *;
+
+.nav-bar {
+  display: flex;
+  justify-content: space-between;
+  font-size: 15px;
+}
+
+.left,
+.center,
+.right {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  div {
```

```suggestion
  align-items: center;
  
  div {
```

scope 달라질때 한줄 띄면 가독성이 올라가

---

완!